code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -50,4 +35,14 @@
 			"<span class='danger'>[src] devours [L]!</span>",
 			"<span class='userdanger'>You feast on [L], restoring your health!</span>")
 		adjustBruteLoss(-L.maxHealth/2)
-		L.gib()
\ No newline at end of file
+		L.gib()
+
+/mob/living/simple_animal/hostile/megafauna/ex_act(severity, target)
+	switch(severity)
+		if(1)
+			adjustBruteLoss(600)
+		if(2)
+			adjustBruteLoss(300)
```

Comment:
Minibombs are traitor gear. IEDs are weak and hard to aim. It's usually more simple to get three more shots with KA than to bother with grenades at all.